bootstrap.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# coding=utf-8
+import argparse
+import os
+import subprocess
+from string import Template
+
+from blessings import Terminal
+
+
+DEFAULT_DIRECTORY_NAME = 'mozilla-payments'
+GIT_REPOS = [
+    {'org': 'Kinto', 'name': 'kinto'},
+    {'org': 'mozilla', 'name': 'payments-example'},
+    {'org': 'mozilla', 'name': 'payments-service'},
+    {'org': 'mozilla', 'name': 'payments-ui'},
+    {'org': 'mozilla', 'name': 'solitude'},
+    {'org': 'mozilla', 'name': 'solitude-auth'},
+]
+PAYMENT_ENV_DIR = os.path.abspath(os.path.dirname(__file__))
+LOCAL_ENV = Template((os.fdopen(os.open(
+    os.path.join(PAYMENT_ENV_DIR, 'env', 'local.env.dist'),
+    os.O_RDWR), "r+").read()))
+SEPARATOR = """
+###############################################################################
+💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰
+###############################################################################"""
```

Comment:
So some of the failures on pep8 in the file are string length in these docstrings, and fixing them up makes the code significantly less readable.

If you're cool with it I'll fix all line length issues that aren't in docstrings and all other issues period.